### PR TITLE
Create microservice for multisignature pool - Closes #159

### LINKED ISSUE #159: Create microservice for multisignature pool
### Description

Create a new microservice from a template. Name it `transaction`.

### Motivation

A separate microservice helps with the separation of functional parts of the application. It is easy to distinguish between passive and active parts of the application.

### Acceptance Criteria

- New microservice is ready: starts and registers in the gateway properly
- New microservice is mentioned in the readme file

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #159 

### How was it solved?
- [x] New microservice (`transaction`) is ready: starts and registers in the gateway properly
- [x] New microservice is mentioned in the readme file
- [x] Update readme for newsfeed and transaction service

### How was it tested?
Local + Jenkins

### CODE REVIEW

File: services/transaction/config.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = {};
+
+// Moleculer broker config
+config.transporter = process.env.SERVICE_BROKER || 'redis://localhost:6379/0';
+config.brokerTimeout = Number(process.env.SERVICE_BROKER_TIMEOUT) || 5; // in seconds
```
Comment (sameersubudhi) [inline services/transaction/config.js:20]: Please update the config to keep consistency across all the microservices. Please make sure to update the template as well.
```suggestion
config.brokerTimeout = Number(process.env.SERVICE_BROKER_TIMEOUT) || 10; // in seconds
```

File: README.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ Each service is an independent part of the repository and is placed in a separat
 | [Gateway](services/gateway) | The Gateway provides the API, which all users of Lisk Service can access and use. Its main purpose is to proxy API requests from users to other services provided by Lisk Service. This provides the users with a central point of data access that never breaks existing application compatibility.|
 | [Lisk](services/core) | The Lisk Core service acts as a bridge between the Lisk Core and the Lisk Service API. Its main purpose is to provide enriched data from the Lisk Core API. This service is aimed at providing high availability, and both efficient and reliable access to the Lisk Core API. |
 | [Market](services/market) | The Market service allows price data retrieval. It supports multiple sources to keep the current Lisk token price up-to-date and available to the clients. |
+| [Newsfeed](services/newsfeed) | The news feed service is a one-purpose microservice that polls the content sharing platforms and shares collected data with UI clients such as Lisk Desktop |
```
Comment (sameersubudhi) [inline README.md:24]:
```suggestion
| [Newsfeed](services/newsfeed) | The Newsfeed service is a single-purpose microservice that polls the content sharing platforms and shares collected data with UI clients such as Lisk Desktop |
```

File: README.md [hunk 3]
```diff
@@ -21,6 +21,8 @@ Each service is an independent part of the repository and is placed in a separat
...
+| [Transaction](services/transaction) | The Transaction service is a proxy between all participants before the multisignature transaction is ready to broadcast |
```
Comment (sameersubudhi) [inline README.md:25]:
```suggestion
| [Transaction](services/transaction) | The Transaction service is a proxy between all participants before a multisignature transaction is ready to be broadcast to the blockchain |
```

File: services/newsfeed/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# Lisk Service Template
+# Lisk Service Newsfeed
 
-The Template service is an abstract service that all of Lisk Service services are inherited from. It allows all services to share a similar interface and design pattern. Its purpose is to reduce code duplication and increase consistency between each service, hence simplifying code maintenance and testing.
+The news feed service is a one-purpose microservice that polls the content sharing platforms and shares collected data with UI clients such as Lisk Desktop
```
Comment (sameersubudhi) [inline services/newsfeed/README.md:3]:
```suggestion
The Newsfeed service is a single-purpose microservice that polls the content sharing platforms and shares collected data with UI clients such as Lisk Desktop
```

File: services/transaction/README.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+# Lisk Service Transaction
+
+The Transaction service is a proxy between all participants before the multisignature transaction is ready to broadcast
```
Comment (sameersubudhi) [inline services/transaction/README.md:3]:
```suggestion
The Transaction service is a proxy between all participants before a multisignature transaction is ready to be broadcast to the blockchain
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Please consider the following:

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Review by MichalTuleja — APPROVED** [review]
LGTM!

**Pull request merged**